code/modules/mob/living/living.dm

```diff
@@ -412,6 +412,9 @@ default behaviour is:
 	var/datum/reagents/ingested_reagents = get_ingested_reagents()
 	if(ingested_reagents)
 		ingested_reagents.clear_reagents()
+	var/datum/reagents/inhaled_reagents = get_inhaled_reagents()
+	if(inhaled_reagents)
```

Comment:
Won't block for this, but it might be worth doing something with a list and |= to avoid clearing the same reagent holders repeatedly, as the getters on /living return the same holder.